chain/client/src/sync/state.rs

```diff
@@ -535,61 +520,33 @@ impl StateSync {
     /// Only select candidates that we have no pending request currently ongoing.
```

Comment:
Is the comment still up to date? The part about validators being candidates in particular.

Reply:
Thanks, removed the comment about the validators.